tequilapi/endpoints/identities.go

```diff
@@ -0,0 +1,39 @@
+package endpoints
+
+import (
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/identity"
+	"github.com/mysterium/node/service_discovery/dto"
+	"github.com/mysterium/node/tequilapi/utils"
+)
+
+type IdentityData struct {
```

Comment:
- `IdentityData` -> `IdentityDto`
- `Identities ` -> `IdentityListDto`
- DTO structures does not need to be public